packages/devextreme/js/__internal/ui/overlay/m_overlay_position_controller.ts

```diff
@@ -200,86 +276,111 @@ class OverlayPositionController {
 
   _renderContentInitialPosition(): void {
     this._renderBoundaryOffset();
+
     resetPosition(this._$content);
-    // @ts-expect-error ts-error
-    const wrapperOverflow = this._$wrapper.css('overflow');
+
+    const wrapperOverflow = this._$wrapper?.css('overflow') ?? '';
+
     this._$wrapper?.css('overflow', 'hidden');
 
-    if (!this._props._skipContentPositioning) {
+    if (!this._properties._skipContentPositioning) {
       const resultPosition = positionUtils.setup(this._$content, this._position);
+
       this._initialPosition = resultPosition;
     }
-    // @ts-expect-error ts-error
+
     this._$wrapper?.css('overflow', wrapperOverflow);
+
     this.detectVisualPositionChange();
   }
 
-  _raisePositionedEvents(event): void {
+  _raisePositionedEvents(event?: PointerLikeEvent): void {
     const previousPosition = this._previousVisualPosition;
     const newPosition = this._visualPosition;
 
-    const isVisualPositionChanged = previousPosition?.top !== newPosition.top
-            || previousPosition?.left !== newPosition.left;
+    const isTopEqual = previousPosition?.top === newPosition?.top;
+    const isLeftEqual = previousPosition?.left === newPosition?.left;
+
+    const isVisualPositionChanged = !(isTopEqual && isLeftEqual);
```

Comment:
I believe this is more clear:
```suggestion
    const isVisualPositionChanged = !isTopEqual || !isLeftEqual;
```

Reply:
I agree partially; in any case, we use this logical expression everywhere in the code.